lib/engine/game/g_1825/map.rb

```diff
@@ -544,6 +560,17 @@ def game_tiles
           append_game_tiles(gtiles, K5_TILES) if @kits[5]
           append_game_tiles(gtiles, K6_TILES) if @kits[6]
           append_game_tiles(gtiles, D1_TILES) if @optional_rules.include?(:d1)
+          gtiles = db1_tiles(gtiles) if @optional_rules.include?(:db1)
+          gtiles
```

Comment:
why do you return g_tiles here?

Reply:
why do you reassign gtiles? you mutate it in db1_tiles?

Reply:
I reassign it because the way everything even function arguments is a reference was a bit of a mystery to me. Fixed.

I return gtiles because that's what game_tiles did before I started messing around with it.